### PR TITLE
Add & remove

### PR DESCRIPTION
**Pull request by davidigbo**
In this pull request, I create how to add and remove item from to-do-list application by javascript functionality.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @david30-maker,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has no customized README. Kindly make sure your Readme following these rules: No empty sections, no broken links, no related texts with the project, no templated comments, microverse logo should delete and all sections should be added. Only the FAQ section is optional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    index: 2,
+    description: 'add task todo list',
+    completed: false,
+  },
+  {
+    index: 3,
+    description: 'remove todo list',
+    completed: false,
+  },
+];
```
Comment (hiromataba) [inline src/index.js]: - [ ] You should remove all the hardcoded  array of items because now you can add and remove

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  const todoList = document.getElementById('todo-list');
+
+  tasks.forEach((item) => {
+    const listItem = document.createElement('li');
+    listItem.innerHTML = `
+      <input class="checkbox" type="checkbox" name="" value="">
+      <span>${item.description}</span>
+      `;
+    todoList.appendChild(listItem);
+  });
```
Comment (hiromataba) [inline src/index.js]: -  [ ]. I am not able to add an item please fix that and this prevents me from check the other functionalities of the project so the next reviewer will have right to check them as well



https://user-images.githubusercontent.com/75126481/222556034-21e7b879-ae9c-4825-aeb4-86c6ff17c4da.mov

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @david30-maker    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (davidigbo) [reply src/index.js]: @hiromataba, I think it is nex stage of the project that one can add all interactiveness and functionality required, please check the requirement well. Thanks

Comment (davidigbo) [reply src/index.js]: @hiromataba, Thanks, I got a better understand👌👍❤️

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @david30-maker :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 
- All functions adding/editing/deleting well :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (hiromataba) [reply src/index.js]: Glad to hear that 🙂 @david30-maker

**Pull request merged**